test/randomized_benchmarking/test_randomized_benchmarking.py

```diff
@@ -0,0 +1,294 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Test for randomized benchmarking experiments."""
+
+from test.base import QiskitExperimentsTestCase
+
+import numpy as np
+from ddt import ddt, data, unpack
+from qiskit.circuit.library import SXGate, CXGate, TGate, XGate
+from qiskit.exceptions import QiskitError
+from qiskit.providers.aer import AerSimulator
+from qiskit.providers.aer.noise import NoiseModel, depolarizing_error
+from qiskit.quantum_info import Clifford
+from qiskit.quantum_info.operators.predicates import matrix_equal
+
+from qiskit_experiments.library import randomized_benchmarking as rb
+
+
+class RBTestCase(QiskitExperimentsTestCase):
+    """Base test case for randomized benchmarking defining a common noise model."""
+
+    def setUp(self):
+        """Setup the tests."""
+        super().setUp()
+
+        # depolarizing error
+        self.p1q = 0.002
+        self.p2q = 0.010
+        self.pvz = 0.0
+
+        # basis gates
+        self.basis_gates = ["sx", "rz", "cx"]
+
+        # setup noise model
+        sx_error = depolarizing_error(self.p1q, 1)
+        rz_error = depolarizing_error(self.pvz, 1)
+        cx_error = depolarizing_error(self.p2q, 2)
+
+        noise_model = NoiseModel()
+        noise_model.add_all_qubit_quantum_error(sx_error, "sx")
+        noise_model.add_all_qubit_quantum_error(rz_error, "rz")
+        noise_model.add_all_qubit_quantum_error(cx_error, "cx")
+
+        self.noise_model = noise_model
+
+        # Need level1 for consecutive gate cancellation for reference EPC value calculation
+        self.transpiler_options = {
+            "basis_gates": self.basis_gates,
+            "optimization_level": 1,
+        }
+
+        # Aer simulator run options
+        self.run_options = {
+            "seed_simulator": 123,
+            "noise_model": self.noise_model,
+        }
+
+        self.backend = AerSimulator()
+
+    def is_identity(self, circuits):
```

Comment:
Since this method is calling assert, it should probably be named accordingly like `assertAllIdentity(circuits)` or. Or change it to `all_identity` and return type to `bool` and call it like:
```
self.assertTrue(self.all_identity(exp.circuits()))
```

Reply:
added `assertAllIdentity`